devnotes/documentation.md

```diff
@@ -22,13 +22,17 @@ environment, ensure that:
 In order to start from a common point, flash the original OpenPOWER firmware
 for Talos II.
 
+0. Checkout Talos II in [snipeit](http://snipeit) to avoid conflicts when
```

Comment:
@macpijan Should snipeit be mentioned in this documentation? It is an internal tool after all.

Reply:
@IgorBagnucki It can be if it is needed to work on hardware.